src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/VSResources.resx

```diff
@@ -295,4 +295,7 @@ In order to debug this project, add an executable project to this solution which
     <value>The .xaml file was in an unexpected format, near line {0} column {1}.</value>
     <comment>{0} is the line number of the problematic XAML, and {1} is the column.</comment>
   </data>
+  <data name="InfoBarMessageNuGetCycleDetected" xml:space="preserve">
+    <value>NuGet restore detected a cycle. NuGet Error NU1108</value>
```

Comment:
Can we provide the user more details, or explain the impact of a cycle? We should help the user understand why a cycle in NuGet Restore is bad.

Reply:
I changed this to something more verbose.

_A circular dependency A -> B -> A was detected during NuGet Restore which caused NuGet to stop restoring.
Because the NuGet restore didn't finish correctly, the project might be in a bad state.

Projects might have a package that sets AssetTargetFallback, not supported for this scenario, which caused certain PackageReference to be included.

NuGet Error NU1108_